### ISSUE TITLE
Error message in another language

### ISSUE THREAD

**Issue by juliardi** [go-ozzo/ozzo-validation#57]
Are there any way to define error message in another language? 
Right now we can define it by using custom error message, but it would be redundant when we validate a bunch of value with the same validation rule.

**Comment by juliardi** [comment]
Yes, it would be good to centralize the error messages in a struct or map

**Comment by AleksandrGreen** [comment]
As option, you can use message keys instead messages. For example, instead of "must be a valid email address" => "govalidator.isEmail". And you can add default map function in your lib, add possibility to user to pass translation function to validate methods (Validate, ValidateStruct) or just modify package varaiables: func MyMapper(key string) string.

```
var mySuperMap := string[string]{
 //Fill map here
 "ozzo.required" => "darf nicht leer sein",
 "govalidator.isUUIDv4" => "muss eine gültige UUID v4 sein"
}

v.Validate(params, v.Field(&params.CaseID, v.Required, is.UUIDv4), func(key string) string {
  return mySuperMap[key]
})
```
Of course this example is simplified.

**Comment by qiangxue** [comment]
A workaround is to create a rule,  customize its error message, and save it in a variable. You can then reuse this rule variable without redoing the message customization.

Still, like the string map solution, it doesn't fully solve the problem when a message needs to be chosen based on the "current" language.

**Comment by qiangxue** [comment]
This has been discussed in great details and implemented in #91.
